test/Infrastructure/LeanCode.Kratos.Tests/KratosAuthenticationHandlerTests.cs

```diff
@@ -191,14 +227,15 @@ public async Task Maps_session_and_identity_properties_to_principals_claims()
                     c.Add(new(o.RoleClaimType, "developer"));
                 }
 
-                if (s.Identity.MetadataPublic is MetadataPublic { IsAdmin: true })
+                if (s.Identity.MetadataPublic is Option<object> { Value: MetadataPublic { IsAdmin: true } })
```

Comment:
This is quite disgusting, we now have to match against the `Option<object>` intermediate thingy.